tests/settings/ConfigurationGatheringTest.py

```diff
@@ -182,7 +182,7 @@ def test_back_saving(self):
             lambda *args: True,
```

Comment:
Wanted to use `ConfigurationGatheringTest: Use StringProcessing.escape` for the commit message, but it would be too long.